code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -382,6 +382,30 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
 			items_to_spawn = list(/obj/item/weapon/reagent_containers/food/snacks/mint/syndiemint/homemade = 50)
 	..()
 
+/obj/item/weapon/storage/pill_bottle/mint/
```

Comment:
no trailing / on the abstract type
also, the name and description seem to have multiple typos, but that might be the joke...?